jsapp/js/components/modalForms/projectSettings.es6

```diff
@@ -822,7 +823,7 @@ class ProjectSettings extends React.Component {
                 maxLength={NAME_MAX_LENGTH}
                 id='name'
                 placeholder={t('Enter title of project here')}
-                value={this.state.name}
+                value={removeInvalidChars(this.state.name)}
```

Comment:
Instead of using it here, please use it in `onNameChange` and `onDescriptionChange` (similarly as in `editableForm.es6` below) :heart: